docs/proton-kafka.md

```diff
@@ -559,19 +558,28 @@ CREATE EXTERNAL STREAM example (
     _tp_message_headers map(string, string)
 ) settings type='kafka',...;
 ```
-Then insert data to the external stream via `INSERT INTO` or materialized views, with a map of string pairs as custom headers for each message.
 
-## DROP EXTERNAL STREAM
+Then insert rows to the external stream via `INSERT INTO` or Materialized Views, the `_tp_message_headers` will be set to the headers of the Kafka message.
+
+#### sharding_expr (Partition Logic)
```

Comment:
you can add {#sharding_expr} in the end, to keep using sharding_expr as the url hash. Other this link will be `/proton-kafka#sharding_expr-partition-logic` and break the link to `/proton-kafka#sharding_expr`